Amsterdam/dso-api
# Yrob/GitHub ci

* Git workflows for:
     - pre-commit hooks
     - pytest
     -  openapi spec validator
     (expand the  "All checks have passed" tab below to see results)

* Use Postgres 14 in compose file
* Local deployment is now as simple as `docker compose up` (migrations and schema import are automatic)
* Remove old Jenkins stuff
